### DISCUSSION TITLE
SDRPlay RSP2 working with dump1090_rs

### DISCUSSION THREAD

**Discussion by cschmittiey** [rsadsb/dump1090_rs#58] [Show and tell]
Hey, just wanted to let y'all know that I got my SDRPlay RSP2 working with dump1090_rs. attached below is the custom config I used. It could probably use more tweaking but I just wanted to at least write in and let you know it works. I set the gain to 4 because that's the default according to the soapy library, but that should probably be tweaked when setting it up.

If you wanted to use an RSP1, you should just be able to take out the antenna configuration, and you might need to tweak the gain settings.
```

[[sdrs]]
driver = "sdrplay"

[[sdrs.gain]]
key = "rfgain_sel"
value = 4.0

[sdrs.antenna]
name = "Antenna B"
```

**Comment by cschmittiey** [comment]
oh and i should of course mention that you need to have the SDRPlay API v3 installed, with https://github.com/pothosware/SoapySDRPlay3 also built and installed.

**Comment by wcampbell0x2a** [comment]
Cool! Glad it worked out for you
